packages/cursorless-vscode/src/ide/vscode/hats/VscodeHatRenderer.ts

```diff
@@ -203,8 +203,8 @@ export default class VscodeHatRenderer {
     }
 
     const svg = originalSvg
-      .replace(/fill="[^"]+"/, `fill="${color}"`)
-      .replace(/fill:[^;]+;/, `fill:${color};`);
```

Comment:
How come we dropped this one? Still could be useful

Remember that once your other PR merges, we might get svgs with this way of doing fill

Reply:
See answer above. I don't know what format that would be?

Reply:
```svg
<rect x="10" height="180" y="10" width="180" style="stroke: black; fill: red;"/>
```